app/code/Magento/PageCache/view/frontend/layout/default.xml

```diff
@@ -6,10 +6,15 @@
  */
 -->
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <head>
+        <script src="Magento_PageCache/js/inline-form-key-provider.js"/>
+    </head>
     <body>
         <referenceBlock name="head.components">
             <block class="Magento\Framework\View\Element\Js\Components" name="pagecache_page_head_components" template="Magento_PageCache::js/components.phtml"/>
         </referenceBlock>
+        <referenceBlock name="head.additional">
```

Comment:
Seems like this block could be removed. Could you do that?

Reply:
sure thing. accidentally missed it

Reply:
moved logic from head to head.additional block to load asynchronously, won't remove